src/clr-angular/button/button-loading/loading-button.ts

```diff
@@ -59,30 +59,38 @@ export class ClrLoadingButton implements LoadingListener {
 
     constructor(private el: ElementRef, private renderer: Renderer2) {}
 
-    updateLoadingState(state: ClrLoadingState): void {
+    loadingStateChange(state: ClrLoadingState): void {
         this.state = state;
 
         switch (state) {
             case ClrLoadingState.DEFAULT:
+                this.renderer.removeStyle(this.el.nativeElement, "width");
```

Comment:
Why did you remove this? Does it mean the button never reverts to pure dynamic width?